### PR TITLE
Create Italian Translation

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+"Last-Translator: Albano Battistella <albanobattistella@gmail.com>\n"
+"Language-Team: Italian <LL@li.org>\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: de.swsnr.turnon.desktop.in:3 resources/de.swsnr.turnon.metainfo.xml.in:5
+msgid "Turn On"
+msgstr "Turn On"
```
Comment (swsnr) [inline po/it.po:21]: The question is still why you can't translate the app name?

Comment (albanobattistella) [reply po/it.po:21]: Hi,
There are App names that are not translatable in Italian or other languages.
This is one of those cases.

Comment (swsnr) [reply po/it.po:21]: How come?  As said, I've perfectly been able to translate it to German...  "(to) turn on (a device)" is not some kind of coinage, after all, it's just a regular verb :shrug:

Comment (swsnr) [reply po/it.po:21]: I'm just being curious; if it's all that important that the app name isn't translated in Italian, then so be it :shrug:

**Comment by swsnr** [comment]
@albanobattistella I've set up weblate for translations at <https://translate.codeberg.org/projects/de-swsnr-turnon/de-swsnr-turnon/> and directly uploaded your translation from this PR, so I'm closing it now.

If you have a codeberg account, please do feel free to tell me, and I can add you as a translator for Italian.

**Comment by swsnr** [comment]
@albanobattistella I've released 1.2.0 including your translation.  It should've found its way to flathub by tomorrow; you may want to test your translation.  If it does not work please do file an issue here.

**Comment by swsnr** [comment]
So, [flathub broke themselves](https://github.com/flathub/flathub/issues/5806), so it's probably not going to be tomorrow...

**Pull request closed without merging**